config/environments/development.rb

```diff
@@ -58,4 +56,17 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.action_mailer.delivery_method = :letter_opener
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.default_url_options = { host: ENV['MAIL_HOST'] }
+  # config.action_mailer.delivery_method = :smtp
```

Comment:
No need to of this setting as we changed   # config.action_mailer.delivery_method = :smtp
 to   config.action_mailer.delivery_method = :letter_opener
comment or remove this